packages/ui-extensions/src/surfaces/admin/api/purchase-options-card-action/purchase-options-card-action.ts

```diff
@@ -0,0 +1,10 @@
+import type {StandardApi} from '../standard/standard';
+import type {ExtensionTarget as AnyExtensionTarget} from '../../extension-targets';
+
+export interface PurchaseOptionsCardConfigurationApi<
+  ExtensionTarget extends AnyExtensionTarget,
+> extends StandardApi<ExtensionTarget> {
```

Comment:
I think you will want to extend `ActionExtensionApi` here, or just add the `close` method from that type (I forget if `extends` can override a property, in this case `data`.

Reply:
Correct, changed.